scss/detail.scss

```diff
@@ -0,0 +1,28 @@
+* {
+  font-family: $font-family;
+}
+
+h1 {
+  font-size: 2rem;
```

Comment:
We prefer use px because our designers use it as a basis for creating components and mockups.
But i think it's okay to use rem if the design is keeping. ☺️